packages/hardhat/deploy/00_deploy_your_contract.ts

```diff
@@ -1,26 +1,34 @@
 import { HardhatRuntimeEnvironment } from "hardhat/types";
 import { DeployFunction } from "hardhat-deploy/types";
 
+/**
+ * Deploys a contract named "YourContract" using the deployer account and
+ * constructor arguments set to the deployer address
+ *
+ * @param hre HardhatRuntimeEnvironment object.
+ */
 const deployYourContract: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
   const { deployer } = await hre.getNamedAccounts();
   const { deploy } = hre.deployments;
 
   await deploy("YourContract", {
     from: deployer,
-    // Constructor args.
-    // Use your frontend address
+    // By default the deployer account will be one that comes with Hardhat, which anyone can access.
+    // You can use your frontend address as the deployer account to run the deploy script.
+    // The deployer account should have sufficient balance to pay for the gas fee for contract creation.
```

Comment:
```js
// The deployer account should have sufficient balance to pay for the gas fee for contract creation.
```

I think this line should go above line 15 (`from: deployer`), since it's there where we are setting the deployer account to deploy the contract. On `args: [deployer]` we are just passing constructor arguments on deploy, and can be any address (no balance needed!)

Maybe also mention:
- it's only needed for live networks (no need to worry about balance when in hardhat)
- The `yarn account` command.

Quick attempt (feel free to change anything):
```js
// In live networks, the deployer account should have sufficient balance to pay for the gas fee for contract creation. You can run the `yarn account` command to check your balance.
```

Reply:
thank you! added this and also the yarn generate to create a new account (this is also in the readme but added it briefly on here) can make any other change needed as well